README.md

```diff
@@ -226,7 +227,25 @@ said apparatus comprising a base, a spring member coupled to the base, and a cag
 ...
 ```
 
-The metadata will list the custom namespacse and correctly apply it to the annotations of the property.
+The metadata will list the custom namespace and correctly apply it to the annotations of the property.
+
+## Compatibility
+By default SciKGTeX overwrites the PDF metadata catalog and is not fully compatible with other metadata specification packages such as hyperxmp or pdfx.
+On top of that, the generated metadata currently can not comply with the PDF/A standard because of the user-defined properties.
+When submitting to conferences or other subsequent processors data written in the standard PDF metadata field is often erased.
+To prevent this use the SciKGTeX compatibility mode.
+
+The compatibility mode can be activated by importing the package with the compatibility parameter:
+
+```
+\usepackage[compatibility]{scikgtex}
```

Comment:
The name was pdfa before... I think compatibility fits better but is a little long... Other suggestions?

Reply:
Sounds good to me.